src/Symfony/Component/AssetMapper/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+7.3
+---
+
+ * Make `asset-map:compile` remove assets in debug mode
+ * Add `--force` to the `asset-map:compile` command to make it generate assets in debug mode
```

Comment:
It would not have been easier to get / use with an option "--purge" + keeping the current behaviour ? 

A compile command that clear files feels very weird .. using different arguments to do the same thing also 🤷

Reply:
My reasoning is that in dev mode you don't want the assets to block dynamic generation, yet this is the default right now, and this serves no DX purpose.

Reply:
Current DX can be improved for users after they compile assets locally, _**yes**_.  

But this PR  _can_ also make things worse for all the others..  Two _**opposite**_ behaviour between debug and not debug..  i'm not sure this will be a DX win. 🤷‍♂️ 

Two commands: `asset-map:compile` and `asset-map:purge` (or `asset-map:clear`) feels easier to explain, document, use. 

Just a personal feeling.

Reply:
Please explain what you mean by "worse".
The behavior I'm proposing here is not opposite, it's consistent with what each mode (debug/non-debug) needs to work. Adding a myriad of commands for fine grained operations will just make things more complex, aka worse, when we know what the behavior should be to make things Just Work (TM)